ouroboros-consensus-diffusion/src/ouroboros-consensus-diffusion/Ouroboros/Consensus/Node/Tracers.hs

```diff
@@ -81,7 +81,7 @@ instance (forall a. Semigroup (f a))
       , blockchainTimeTracer          = f blockchainTimeTracer
       , forgeStateInfoTracer          = f forgeStateInfoTracer
       , keepAliveClientTracer         = f keepAliveClientTracer
-      , consensusStartupErrorTracer   = f consensusStartupErrorTracer
+      , consensusErrorTracer          = f consensuErrorTracer
```

Comment:
I think there is a typo here: "consensu"

Reply:
Fixed.